### DISCUSSION TITLE
Questions about read-only Settings

### DISCUSSION THREAD

**Discussion by fanzhongxiang** [WinMerge/winmerge#2363] [Q&A]
Hi, I am a new user of winmerge, and I want to know if I choose read-only when selecting a folder, what will be the impact? Will not select read-only cause the file to be modified when just simply comparing?

**Comment by sdottaka** [comment]
Whether the read-only option is selected or not, a compare operation will not update files.
If the read-only option is selected, operations that would update the files, such as edit operations and merge operations, will be disabled.
